src/datasets/io/parquet.py

```diff
@@ -12,6 +12,35 @@
 from .abc import AbstractDatasetReader
 
 
+PARQUET_ROW_GROUP_SIZE_FOR_AUDIO_DATASETS = 100
+PARQUET_ROW_GROUP_SIZE_FOR_IMAGE_DATASETS = 100
+PARQUET_ROW_GROUP_SIZE_FOR_BINARY_DATASETS = 100
+
+
+def get_writer_batch_size(ds_config_info: DatasetInfo) -> Optional[int]:
+    """
+    Get the writer_batch_size that defines the maximum row group size in the parquet files.
+    The default in `datasets` is 1,000 but we lower it to 100 for image datasets.
+    This allows to optimize random access to parquet file, since accessing 1 row requires
+    to read its entire row group.
+    Args:
+        ds_config_info (`datasets.info.DatasetInfo`):
+            Dataset info from `datasets`.
+    Returns:
+        writer_batch_size (`Optional[int]`):
+            Writer batch size to pass to a dataset builder.
+            If `None`, then it will use the `datasets` default.
+    """
+    if "Audio(" in str(ds_config_info.features):
+        return PARQUET_ROW_GROUP_SIZE_FOR_AUDIO_DATASETS
+    elif "Image(" in str(ds_config_info.features):
+        return PARQUET_ROW_GROUP_SIZE_FOR_IMAGE_DATASETS
+    elif "'binary'" in str(ds_config_info.features):
+        return PARQUET_ROW_GROUP_SIZE_FOR_BINARY_DATASETS
+    else:
+        return None
```

Comment:
I think we should traverse the features here instead of checking the strings' presence.